docs/recipes/testing-with-selenium-webdriverjs.md

```diff
@@ -0,0 +1,131 @@
+# Setting up AVA with Selenium WebDriverJS
+
+This recipe shows you how to use the Selenium WebDriverJS (official JavaScript implementation) with AVA to test web apps.
+
+## Setup
+
+This recipe uses the following packages:
+
+1. [selenium-webdriver](https://www.npmjs.com/package/selenium-webdriver)
+2. [chromedriver](https://www.npmjs.com/package/chromedriver)
+
+Install them with:
+
+```console
+$ npm install selenium-webdriver chromedriver
+```
+
+As part of this recipe, we will use Selenium to verify web searches on [Bing](https://www.bing.com) and [Google](https://www.google.com)
+
+## Test files
+
+Create the following files:
+
+- `./test/bingtest.js`
+- `./test/googletest.js`
+
+In both files, let's first include the packages:
+
+```js
+import test from 'ava';
+import {Builder, By, Key, until} from 'selenium-webdriver';
+
+require('chromedriver');
+```
+
+In the `bingtest.js` file, add the following code, which tests whether searching for `webdriver` on Bing, returns results.
+
+```js
+test('Bing Search', async t => {
+	let keyword = "webdriver";
```

Comment:
Use `const` everywhere. And use single-quotes.

Reply:
done